public/index.html

```diff
@@ -1,12 +1,14 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <script src="https://use.fontawesome.com/cb5b364a9a.js"></script>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700&display=swap" rel="stylesheet">
```

Comment:
I've been thinking about what font to use.  I'm gonna steal this!

Reply:
@jsmithb117 please do! From what I saw, this is the one they use on their site.